runbotics/runbotics-desktop/src/action/browser/browser.action-handler.ts

```diff
@@ -105,6 +105,10 @@ export default class BrowserActionHandler extends StatefulActionHandler {
         // process.env['PATH'] = process.env['PATH'] + ':' + process.env['CFG_CHROME_DRIVER'];
         //
         // let driver = await new Builder().forBrowser('chrome').build();
+        if (!this.session) {
+            throw new Error('The browser is not running');
+        }
```

Comment:
let's move it to the private function and include it at the top of other actions or do it in the switch clause. See sap.action-handler.ts for example